.github/workflows/build-windows.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build on Windows
+run-name: Build on Windows
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - '**.rs'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
```

Comment:
Technically we only build `tsh` on Windows, and tsh doesn't have any rust code. So you can omit the .rs and cargo files.

Reply:
Fixed.